spec/controllers/api/v4/categories/runs_controller_spec.rb

```diff
@@ -6,7 +6,7 @@
       let(:category) { create(:category, :with_runs) }
       subject { get :index, params: {category: category.id} }
 
-      it 'retruns a 200' do
```

Comment:
Wow did you catch this, or did Rubocop? Either way, amazing

Reply:
I actually saw it while clicking through the different files and noticed that something was changing even though the wording looked identical XD